src/config.rs

```diff
@@ -345,6 +355,16 @@ fn parse_xpub(s: &str) -> Result<(XyzPubKey, RescanSince)> {
     Ok((xpub, rescan))
 }
 
+fn parse_descriptor(s: &str) -> Result<(String, RescanSince)> {
+    let mut parts = s.splitn(2, ':');
```

Comment:
Could descriptors contains `:`? I can't seem to find any descriptors that do, but `:` appears to be included in the [`INPUT_CHARSET` used for checksumming](https://github.com/bitcoin/bitcoin/blob/eac65d99ddb27061e6ae0f9448887db86e2c0734/test/functional/test_framework/descriptors.py#L9).

Reply:
`:` separates the descriptor and the "rescan since" timestamp. I just copied `parse_xpub` ([link](https://github.com/shesek/bwt/blob/49736aca1903d5e6b258198d58fe1541766bb826/src/config.rs#L341)).

Reply:
Yes, I understood, I asked whether descriptors can contain `:` because if they do, the `:` separator would be ambiguous with the syntax of the descriptor itself.

But AFAICT, descriptors cannot contain `:`, so this shouldn't be ambiguous.